raptor/raptor.py

```diff
@@ -6,43 +6,48 @@
 
 from __future__ import absolute_import
 
+import json
 import os
 import sys
 
 from mozlog import commandline, get_default_logger
+from mozprofile import create_profile
```

Comment:
Where is the source for create_profile? I can't seem to find it in:

https://searchfox.org/mozilla-central/source/testing/mozbase/mozprofile/mozprofile

Or is it inherited somewhere?

Reply:
Oh wait NVM it's in your vendor'ed custom mozbase below haha ok cool

Reply:
Yeah, no worries :). I have mirror patches in mozilla-central clone for all the mozbase changes, so once we go to vendor it'll be easier to get the mozbase changes landed first.